include/multipass/memory_size.h

```diff
@@ -39,9 +39,12 @@ class MemorySize
     long long in_megabytes() const noexcept;
     long long in_gigabytes() const noexcept;
 
-    std::string human_readable() const;
+    std::string human_readable(unsigned int precision = 1) const;
+
+    static MemorySize from_bytes(long long bytes) noexcept;
```

Comment:
Very good choice. The static function with this name makes it clear what the unit is, which would no be the case with a constructor :+1:

Reply:
Just bringing some Rust habits into C++ :wink: